hack/gardener-integration-test.sh

```diff
@@ -17,7 +17,7 @@ function run-tests-with-git-image () {
     make ginkgo
     make deploy
     hack/deploy-istio.sh
-    ${GINKGO} run --tags istio --label-filter="integration" test/integration/istio
+    ${GINKGO} run --junit-report=junit-report --tags istio --label-filter="integration" test/integration/istio
```

Comment:
Isn't the `.xml` extension missing here? I saw you are using it everywhere else, and it looks a bit inconsistent.